src/memkind_memtier.c

```diff
@@ -626,6 +627,8 @@ builder_movement_create_memory(struct memtier_builder *builder)
 
     create_memory_common(builder, memory);
 
+    memkind_mtt_create();
```

Comment:
the prefix was supposed to be mtt_allocator_, but maybe I did not explicitly said that

also, please check other comments about support for memory builder